src/Pages/Login/login.js

```diff
@@ -1,9 +1,130 @@
 import React, { Component } from 'react';
+import '../Login/Login.scss';
+import Naver from '../../Img/icon-naver.png';
+import Kakao from '../../Img/icon-kakao.png';
+import Google from '../../Img/icon-google.png';
 
-class login extends Component {
+class Login extends Component {
+  constructor() {
+    super();
+    this.state = {
+      // name: '',
+      id: '',
+      pw: '',
+      // email: '',
+      // date_birth: '',
+      // phone_number: '',
+      // borderBottom: '',
+      // borderAll: '',
+    };
+  }
+  handleInput = e => {
+    const { name, value } = e.target;
+    this.setState({ [name]: value });
+    console.log(e.target.value);
+  };
+
+  goToMain = e => {
+    e.preventDefault();
+    fetch('http://10.58.58.131:8000/members/signin', {
+      method: 'POST',
+      body: JSON.stringify({
+        account: this.state.id,
+        password: this.state.pw,
+        // email: this.state.email,
```

Comment:
api 주소의 경우 src/config.js 등의 경로에 모아놓는 것이 유지보수 차원에서 좋습니다. 만약 ip 주소가 바뀐다면 해당 주소가 쓰인 모든 컴포넌트를 돌아다니면서 하나하나 바꿔줘야겠죠? 다음 예시 참고해주세요
```suggestion
    fetch(`${API_URL}/members/signin`, {
      method: 'POST',
      body: JSON.stringify({
        account: this.state.id,
        password: this.state.pw,
```

Reply:
API주소가 정말 자주 바뀌는데 바뀔때 마다 프론트 쪽에서 src/config.js 이곳에 바뀐주소를 수동적으로 저장해주어야하나요?